bahmnicore-omod/src/main/java/org/openmrs/module/bahmnicore/web/v1_0/resource/BahmniDrugResource.java

```diff
@@ -3,11 +3,11 @@
 import org.openmrs.module.webservices.rest.web.representation.NamedRepresentation;
 import org.openmrs.module.webservices.rest.web.representation.Representation;
 import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
-import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_0.DrugResource2_0;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_12.DrugResource1_12;
```

Comment:
any specific reason we are doing down the version? 2.0->1.12

Reply:
@angshu few of the classes like mappers and resources not moved to emrapi-api-2.2 folder. So had to specifically use openmrs1_12 version of classes.